Packs/Cisco-umbrella-cloud-security/Integrations/Cisco-umbrella-cloud-security/Cisco-umbrella-cloud-security.py

```diff
@@ -68,18 +81,31 @@ def remove_domain(client: Client, **args) -> str:
 
 
 def get_destination_domains(client: Client, **args) -> CommandResults:
-    organizationId = args.get('orgId')
-    destinationListId = args.get('destId')
-    uri = f'/{organizationId}/destinationlists/{destinationListId}/destinations'
+    r = client.get_destinations(args.get('orgId'), args.get('destId'))
 
-    r = client._http_request('GET', uri)
+    results = CommandResults(
+        outputs_prefix="Umbrella.Destinations",
+        outputs_key_field="id",
+        outputs=r.get('data'),
+        readable_output=tableToMarkdown('Domains in Destination List', r.get('data'))
+    )
+
+    return results
 
-    for x in r.get('data'):
-        x['Destination List'] = destinationListId
```

Comment:
same here about the release notes on the breaking change

Reply:
RN: "get_destination_domains: context output r.get('data') instead of r"